fission-core/library/Fission/Internal/Orphanage/RSA2048.hs

```diff
@@ -7,5 +7,7 @@ import qualified System.IO.Unsafe      as Unsafe
 
 import           Fission.Prelude
 
+
+-- Wait. This is weird. How did this end up in this module?
 instance Arbitrary Ed25519.SecretKey where
```

Comment:
Yup that is weird! It's probably me being confused, but the compiler warning is turned off because `{-# OPTIONS_GHC -fno-warn-orphans #-}`. All the more reason for us to just bite the bullet and newtype everything. The more that I think about it, the more I'm in favour of just doing that.

Reply:
Maybe to clarify: let's make this change in this PR. This line _should not_ be in this module!